### PR TITLE
Adds API docs for generating tokens via APIv2

### PR DESCRIPTION
**Pull request by shawngustaw**
Refs #SDE-????

TODO:
- [ ] Set milestone label in pull request (SDE version)
- [ ] Updated changelog.md
- [ ] Completed documentation for the new endpoint.
- [ ] Feature has been merged into SDE develop.

### CODE REVIEW

**Comment by shawngustaw** [comment]
Some things to consider:

Right now in the UI there's no means to revoke another users token. Do we want to maintain this behaviour or do we want this to live under /api/v2/users/{id}/api-token/?

Do we prefer POST/PUT/PATCH for generate/regenerate/revoke, or would we rather have one endpoint that we post {"method": "generate"} to?

File: source/includes/api_tokens.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# API Tokens
+
+## Generate an API Token
+
+```http
+POST /api/accounts/api-token/ HTTP/1.1
```
Comment (funkaoshi) [inline source/includes/api_tokens.md]: I think this should live under the users end point: `/api/v2/users/<user_id>/api-token/`. I think the password reset endpoints are a good example to look at. (Also you were missing the `v2` in the URL regardless.)

**Comment by RouganStriker** [comment]
Do we need separate endpoints for generate and re-generate?

**Comment by shawngustaw** [comment]
@RouganStriker I don't think so. Both are actually hidden behind APIToken.generate_token_for anyways which checks if a token exists and will regenerate if so.

File: source/includes/api_tokens.md (hunk 1 above)
Comment (hanifvirani) [inline source/includes/api_tokens.md]: Yeah, it should live under `/api/v2/users/<user_id>/api-token/`. The other thing to look for is if we can combine this with reset password stuff. Like, `generate_token` and `reset_password` could be different things we pass to a common endpoint.

Comment (hanifvirani) [inline source/includes/api_tokens.md]: Whatever we end up with, I think we should document it. No reason to not provide this as a usable API feature.

File: source/includes/api_tokens.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+    "token": "1234567890abcdefghijklmnopqrstuvwxyz",
+}
+```
+
+This endpoint returns a new API Token.
```
Comment (hanifvirani) [inline source/includes/api_tokens.md]: I would use the word `generate` somewhere in here.

**Comment by funkaoshi** [comment]
Mention change in `changelog.md`.

**Comment by shawngustaw** [comment]
#130

**Pull request closed without merging**